src/controllers/IndexController.php

```diff
@@ -406,7 +435,36 @@ class="fb-main page--registration full-height fb-scroll">
             </fieldset>
             <div class="fb-choose-emblem">
               <h6>{tr('Choose an Emblem')}</h6>
-              <div class="emblem-carousel">{$logos_section}</div>
+              <h6>
+                <a href="#" id="custom-emblem-link">or upload your own</a>
```

Comment:
Good catch! Will go ahead and add the translation wrapper for the various strings that were missed.